### PR TITLE
Fix Author Fields of GET Methods 

### LINKED ISSUE #188: Fixing Author fields of API methods
### Issue Description

Some of the GET methods return authors of papers differently. I should be fixed.

### Steps

- [ ] ...
- [ ] ...
- [ ] ...

### Deadline for the Issue

_No response_

### PR DESCRIPTION
**Pull request by hakanaktas0**
Fixes #188

### CODE REVIEW

File: practice_app/api/views.py [hunk 1]
```diff
@@ -90,7 +90,8 @@ def google_scholar(request):
         number = 3
     else:
         number = int(number)
-
+    print(search)
+    print(number)
```
Comment (uyduranomer) [inline practice_app/api/views.py]: I think you forgot to delete these lines

**Review by uyduranomer — CHANGES_REQUESTED** [review]
There is 2 lines to be deleted and everything else is fine I think. If you fix that this pr will be ready to be merged. The else is works flawless 👌

File: practice_app/api/views.py (hunk 1 above)
Comment (hakanaktas0) [reply practice_app/api/views.py]: fixed

**Review by uyduranomer — APPROVED** [review]
Everything looks cool now. Well done!

**Pull request merged**